src/IBM.Watson.TextToSpeech.v1/TextToSpeechServiceExtension.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using IBM.Cloud.SDK.Core.Http;
+using IBM.Cloud.SDK.Core.Service;
+using IBM.Watson.TextToSpeech.v1.Websockets;
+
+namespace IBM.Watson.TextToSpeech.v1
+{
+    public partial class TextToSpeechService : IBMService, ITextToSpeechService
+    {
+        public WebSocketClient SynthesizeUsingWebSocket(SynthesizeCallback callback, string text, string voice = null, string customizationId = null, string accept = SynthesizeEnums.AcceptValue.AUDIO_OGG_CODECS_OPUS, string[] timings = null)
```

Comment:
does accept need a default value? or if provided with null will the service default?

Reply:
the service provides a default value, I changed the default to null